tests/test_download.py

```diff
@@ -124,22 +124,6 @@ def test_incorrect_snapshot_order_news(self):
             start_snapshot = "2021-10"
             urls = get_common_crawl_urls(start_snapshot, end_snapshot, news=True)
 
-    @pytest.mark.flaky(reruns=5)
```

Comment:
Maybe we just mark it as skip if we want to add it back ? 

@pytest.mark.skip(reason="Skipping till we figure out how to get this to a non flaky state")

Reply:
Thanks! Sounds good to me.